packages/uikit-react-native/src/fragments/createGroupChannelFragment.tsx

```diff
@@ -191,8 +198,17 @@ const createGroupChannelFragment = (initModule?: Partial<GroupChannelModule>): G
     const onPressSendFileMessage: GroupChannelProps['Input']['onPressSendFileMessage'] = useFreshCallback(
       async (params) => {
         const processedParams = await onBeforeSendFileMessage(params);
-        const message = await sendFileMessage(processedParams, onPending);
-        onSent(message);
+        const fileSize = (processedParams.file as File)?.size ?? processedParams.fileSize;
```

Comment:
In React Native, since it is not a web environment, there are no File or Blob types.

`import { FileType } from '../platform/types';`

```suggestion
        const fileSize = (processedParams.file as FileType)?.size ?? processedParams.fileSize;
```

Reply:
Thank you for the helpful information. I've made the corrections!